src/bmi_cfe.c

```diff
@@ -1668,18 +1683,24 @@ static int Get_value_ptr (Bmi *self, const char *name, void **dest)
         *dest = (void*)&cfe_ptr->NWM_soil_params.bb;
         return BMI_SUCCESS;
     }
-    if (strcmp (name, "multiplier") == 0) {
-        cfe_state_struct *cfe_ptr;
-        cfe_ptr = (cfe_state_struct *) self->data;
-        *dest = (void*)&cfe_ptr->NWM_soil_params.mult;
-        return BMI_SUCCESS;
-    }
+    //
+    //if (strcmp (name, "multiplier") == 0) {
+    //    cfe_state_struct *cfe_ptr;
+    //    cfe_ptr = (cfe_state_struct *) self->data;
+    //    *dest = (void*)&cfe_ptr->NWM_soil_params.mult;
+    //    return BMI_SUCCESS;
+    //}
+    
     if (strcmp (name, "Klf") == 0) {
         cfe_state_struct *cfe_ptr;
         cfe_ptr = (cfe_state_struct *) self->data;
-        *dest = (void*)&cfe_ptr->soil_reservoir.coeff_secondary;
+        // LKC: this seems to be a bug, since in init_soil_reservoir cfe_ptr->soil_reservoir.coeff_secondary = cfe_ptr->K_lf; 
```

Comment:
The question here though becomes when is `init_soil_reservoir` called.  If it is called during initialization, then it will have the value of `K_lf` from the initial configuration file, and when a parameter update (`SetValue('K_lf')`) occurs between calls to `Initialize` and `Update`, the soil reservoir won't reflect this.  So in this case, does `init_soil_reservoir` need to be called in the case `Klf` is set by this method?